tests/test_fringes.py

```diff
@@ -287,8 +287,8 @@ def test_multiFringes(self):
         stdAfter = np.nanstd(exp.getImage().getArray())
 
         self.assertLess(medianAfter, medianBefore)
-        self.assertFloatsAlmostEqual(medianAfter, 3002.233, atol=1e-4)
-        self.assertFloatsAlmostEqual(stdAfter, 3549.9375, atol=1e-4)
+        self.assertFloatsAlmostEqual(medianAfter, 3001.312, atol=1e-4)
+        self.assertFloatsAlmostEqual(stdAfter, 3549.949, atol=1e-4)
```

Comment:
What did these values change?

Reply:
These values are related to how the fringe seed value is set.  The tolerance likely should be smaller to more accurately represent the error in this measurement due to changes in seed, but I'm not going to worry about it on this ticket.